programs/launcher-service/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ endif()
 target_include_directories(launcher-service PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
 
 target_link_libraries(launcher-service
-                      PRIVATE appbase version chain_api_plugin chain_plugin http_plugin eosio_chain launcher_service_plugin launcher_service_api_plugin fc Boost::program_options ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
+                      PRIVATE appbase version http_plugin eosio_chain launcher_service_plugin launcher_service_api_plugin fc Boost::program_options ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
```

Comment:
I didn't have a problem with the `chain_plugin` dependency, just the `chain_api_plugin` dependency. By keeping the dependency on `chain_plugin` you can still include the `chain_plugin.hpp` header file in `launcher_service_plugin.cpp` and avoid duplicating the definition for `get_info_results` and `get_abi_results`.

Reply:
should be fixed.